src/Application/ConfigResolver.php

```diff
@@ -73,7 +75,7 @@ public static function resolve(array $config, InputInterface $input): Configurat
         }
 
         if (! isset($config['directory'])) {
-            $config['directory'] = $directory;
+            $config['directory'] = $directories;
```

Comment:
Maybe we should rename $config['directory'] into $config['directories'] as we now pass an array of directories ? 

And perhaps add an 
```
if (isset($config['directory']) {
   $config['directories'] = [$config['directory']];
}
```
for less breaking change ? WDYT ?

Reply:
We can allow breaking changes, as this is for V2 😄 
So maybe just make the breaking change and don't add the extra logic.

Reply:
Yeah I just did with without considering BC. As Oliver said it's for v2 and I think BC is fine. So if there is another matter just let me know.